bin/help

```diff
@@ -1,36 +1,102 @@
 #! /usr/bin/env bash
 
 # DOC: Print descriptions of each bin script.
+# DOC: Use -h for help to show optional usage flags
 
 source bin/lib.sh
 
-missing=""
+function print_help {
+  echo "Usage: 
+  ${0} [options] 
 
-for f in bin/* bin/**/*; do
-  # Files containing library code for scripts have .sh or .py file extension,
-  # don't include those or directories.
-  if [[ -d "${f}" || "${f}" =~ .+\.sh || "${f}" =~ .+\.py ]]; then
-    continue
-  fi
+OPTIONS
+  -h    shows help 
```

Comment:
*Optional, non-blocking*: "shows this help message"

Reply:
done